cds_migrator_kit/rdm/migration/transform/xml_processing/rules/base.py

```diff
@@ -276,11 +279,13 @@ def record_restriction(self, key, value):
 @for_each_value
 def report_number(self, key, value):
     """Translates report_number fields."""
-    report_number = StringValue(value.get("a")).parse()
+    rn = value.get("a") or value.get("9")
+    if not rn:
+        raise IgnoreKey("report_number")
+    report_number = StringValue(rn).parse()
     if report_number:
```

Comment:
yes it is needed, because someone might have put just a space (` `) as a value so the string value will eliminate that (it happens as we saw in 906_p field)

Reply:
I thought the `.parse()` is also stripping these values or we could explicitly `strip` ourselves. I think that would make the code more readable at that point

Reply:
We can address this in a separate issue @kpsherva . I will merge this.